storage/aptosdb/src/state_store/state_store_test.rs

```diff
@@ -189,16 +189,16 @@ fn test_retired_records() {
             (address3, value3.clone()),
         ],
         1, /* version */
-        4, /* expected_nodes_created */
+        3, /* expected_nodes_created */
```

Comment:
@sitalkedia could you pls take a look to see if the change makes sense?

Reply:
Yeah I think it's good.

Reply:
+1. Changes look sane to me.